SWIGGYDEMO_REACT/swiggy-react-app/src/components/itemDetailsAndCart/components/listByCategory/ListbyCategory.js

```diff
@@ -0,0 +1,162 @@
+                ?
+                <div className='items'>
+                    <div className = "searchResult-name">Search Result</div>
+                    {
+                        searchResultItems.length != 0 ?
+                        categoryListForSearch.map( (currentCategory, index) => {
+                            return (
+                                    <div key={index} className="item">
+                                        <div className="name">
+                                            <span className="n">{currentCategory}</span><br />
+                                            <span className="i">{searchResultItems[index].length} items</span>
+                                        </div>
+                                        <div className="listItems">
+                                            <Items data={searchResultItems[index]} cartItems={cartItems} onClickADD={onClickADD} onClickPlus={onClickPlus} onClickMinus={onClickMinus}/>
+                                        </div>
+                                    </div>
+                            );
+                        })
+                        : 
+                        <div className = "searchResultEmpty">Search Result Empty</div>
+                    }
+                    <hr />
+                              
+                    {
+                        categoryListForFilter.map( (currentCategory, index) => {
+                                if (itemsOrderedByCategory[index].length !== 0) {
+                                    return (
+                                        <div id={index + 1} key={index} className="item">
+                                            <div className="name">
+                                                <span className="n">{currentCategory}</span><br />
+                                                <span className="i">{itemsOrderedByCategory[index].length} items</span>
+                                            </div>
+                                            <div className="listItems">
+                                                <Items data={itemsOrderedByCategory[index]} cartItems={cartItems} onClickADD={onClickADD} onClickPlus={onClickPlus} onClickMinus={onClickMinus}/>
+                                            </div>
+                                        </div>
+                                    );
+                                }
+                            })
+                    }
+                </div>
+
+                :
+                <div className='items'>
+                    {
+                        categoryListForFilter.map( (currentCategory, index) => {
+                                return (
+                                        <div id={index + 1} key={index} className="item">
+                                            <div className="name">
+                                                <span className="n">{currentCategory}</span><br />
```

Comment:
className n and i ?? Please use proper name

Reply:
Change className